integration-tests/src/test/java/com/linkedin/kafka/clients/utils/LiKafkaClientsTestUtils.java

```diff
@@ -52,4 +56,15 @@ public static String getRandomString(int length) {
     }
     return stringBuiler.toString();
   }
+
+  public static Map<String, byte[]> convertHeaders(Headers headers) {
+    HashMap<String, byte[]> map = new HashMap<>();
+    for (Header header : headers) {
+      if (map.containsKey(header.key())) {
+        throw new IllegalStateException("Duplicate header found" + header.key());
```

Comment:
Headers allow duplicate keys. As a generic util, I am not sure if we should throw an exception. What do you think?

Reply:
i agree. we should just not allow any dup "_*" headers

Reply:
This is test util -- so it won't be imported into the final jar(IMO). 

I have changed the util to only consider the special headers with "_" prefix.